reverse_proxy.go

```diff
@@ -430,6 +431,10 @@ func httpTransport(timeOut int, rw http.ResponseWriter, req *http.Request, p *Re
 		return wsTransport
 	}
 
+	if txn, ok := rw.(newrelic.Transaction); ok {
```

Comment:
I assume this line should be run only if new relic enabled?

Reply:
Yes, will change it.